tests/e2e/common.go

```diff
@@ -0,0 +1,119 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	apiv1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+	clientgo "k8s.io/client-go/kubernetes"
+
+	"github.com/numaproj/numaplane/internal/util"
+	"github.com/numaproj/numaplane/internal/util/kubernetes"
+	planepkg "github.com/numaproj/numaplane/pkg/client/clientset/versioned/typed/numaplane/v1alpha1"
+)
+
+var (
+	dynamicClient dynamic.DynamicClient
+	testEnv       *envtest.Environment
+	ctx           context.Context
+	cancel        context.CancelFunc
+	suiteTimeout  = 5 * time.Minute
+	testTimeout   = 2 * time.Minute
+
+	pipelineRolloutClient           planepkg.PipelineRolloutInterface
+	isbServiceRolloutClient         planepkg.ISBServiceRolloutInterface
+	numaflowControllerRolloutClient planepkg.NumaflowControllerRolloutInterface
+	kubeClient                      clientgo.Interface
+)
```

Comment:
all these variables need to be in a file not including "_test.go" in the name in order to be referenced from the other files